test/e2e/framework/framework.go

```diff
@@ -194,6 +194,13 @@ func (f *Framework) setup() (cleanUpFunc, error) {
 
 	cleanUpFuncs = append(cleanUpFuncs, cf)
 
+	cf, err = f.CreateRoleBindingFromClusterRole(f.Ns, e2eServiceAccount, "monitoring-alertmanager-edit")
```

Comment:
we want `CreateRoleBindingFromRole()` (which doesn't exist yet) because `monitoring-alertmanager-edit` is a Role.

Reply:
Done